test/powershell/Modules/Microsoft.PowerShell.Utility/NewTemporaryFile.Tests.ps1

```diff
@@ -15,19 +15,100 @@
 #>
 
 Describe "NewTemporaryFile" -Tags "CI" {
+    $defaultTemporaryFileExtension = '.tmp'
 
     It "creates a new temporary file" {
         $tempFile = New-TemporaryFile
+        try
+        {
+            
+            Test-Path $tempFile | Should be $true
+            $tempFile | Should BeOfType System.IO.FileInfo
+            $tempFile.Extension | Should be $defaultTemporaryFileExtension
+        }
+        finally
+        {
+            Remove-Item $tempFile -ErrorAction SilentlyContinue -Force
```

Comment:
Ok. But I will wait with updating the tests for the moment as it is more important to know the final decision first whether the implementation would get accepted or if it requires a fix in corefx first.